entries/a/americafirst.com.json

```diff
@@ -3,9 +3,10 @@
     "domain": "americafirst.com",
     "tfa": [
       "sms",
-      "email"
+      "email",
+      "totp"
```

Comment:
I don't see any mention of TOTP.

Reply:
I don't either, but I set it up this week and am using it.

Reply:
Please attach some screenshots of the MFA setup process, login process, or management screen. Please blur or crop out any personal or sensitive information.